lib/ansible/modules/cloud/packet/packet_device.py

```diff
@@ -86,6 +86,13 @@
       - ID of project of the device.
     required: true
 
+  public_ipv4_subnet_size:
```

Comment:
Might be worth detailing what is done if this isn't set as there is no default specified and it's not a required option.